v1models/transform.go

```diff
@@ -408,3 +420,37 @@ func toV2ResourceOperations(readWrite string, v1ros []ResourceOperation) []model
 	}
 	return v2ros
 }
+
+// convertStartingAddressToOneBased convert startingAddress attribute from zero-based to one-based when transforming from v2 to v1 profile
+// TODO Remove this func when the Modbus connector supports zero-based startingAddress
```

Comment:
it's not a TODO anymore
you should only remove this call in xrt bridge when xrt is upgraded

Reply:
I already removed this call from xrt bridge https://github.com/IOTechSystems/xrt-xpert-bridge/pull/9

Reply:
I mean to remove `\\TODO` here
event if the XRT made change, we will still keep this function for all the v1 Modbus Device Profile

Reply:
Got it. Removed the TODO comment and keep the func call in the xrt bridge.